sources/packages/web/src/components/generic/HorizontalSeparator.vue

```diff
@@ -6,7 +6,7 @@
 
 <style lang="scss" scoped>
 .separator {
-  border: 2px solid #e8e8e8;
+  border: 1px solid #e8e8e8;
```

Comment:
We can update the color to come from global style variable.

Reply:
I will delete this component. We can use the `v-divider` instead.

Reply:
Component removed.